.github/workflows/generate-docs.yml

```diff
@@ -36,6 +36,8 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v3
+        with:
+          token: ${{ secrets.BOT_GITHUB_TOKEN }}
       - name: Checkout ${{ github.event.inputs.source-repo }}
         uses: actions/checkout@v3
         with:
```

Comment:
For the other repo, it doesn't matter which token we use, since we're not writing to it